Countdown/Countdown.js

```diff
@@ -98,6 +100,11 @@ if (remainingDays === 1) {
 }
 postfixText.font = Font.regularSystemFont(20)
 postfixText.textColor = textColor;
+let provider2 = widget.addText(remainingWeeks+ " " + (t[2]))
+provider2.font = Font.mediumSystemFont(16)
+provider2.textColor = textColor
+provider2.centerAlignText()
+textStack.addSpacer()
```

Comment:
- please use a separated horizontal stack with a spacer at beginning and end, and not the one from the remaining days (textStack).
-  Please make this part configurable, because maybe not everyone would like to see also the remaining weeks